### DISCUSSION TITLE
Is there a way to Cherrypick via GitHub URL?

### DISCUSSION THREAD

**Discussion by zard444** [community/community#22408] [New to GitHub]
<p>E.g.</p>
<p>I have 2 commit pull requests from a development branch.</p>
<p>I only want to merge 1 commit.<br>
Ideally prefer to do this from my repository e.g. GitHub URL "<a href="https://github.com/(user)/(repository)/pulls" rel="noopener nofollow ugc">https://github.com/(user)/(repository)/pulls</a>? "<br>
However there doesn’t seem to be any option or documentation showing if it’s possible.</p>
<p>Just wondering if anyone knows if it’s possible?</p>
<p>Thanks!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I’m not aware of any way to do that over the web interface. What you can always do is to <a href="https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally">check out the PR branch locally</a>, and cherry-pick from there. I hope that helps! :smiley_cat:</p>

**Comment by zard444** [comment]
<p>Ok thanks i thought that seemed to be the case, perhaps i’ll get the local version of Github Desktop and do that. Thanks for your input :slight_smile:</p>
